modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -275,83 +276,6 @@ export default {
         this.getPieces();
       }
     },
-    onPreview(id) {
-      this.preview(this.findDocById(this.items, id));
-    },
-    async onArchive(id) {
-      const piece = this.findDocById(this.items, id);
-      if (await this.archive(piece)) {
-        apos.bus.$emit('content-changed');
-      }
-    },
-    async onRestore(id) {
-      const piece = this.findDocById(this.items, id);
-      if (await this.restore(piece)) {
-        apos.bus.$emit('content-changed');
-      }
-    },
-    async onDiscardDraft(id) {
-      const piece = this.findDocById(this.items, id);
-      if (await this.discardDraft(piece)) {
-        apos.bus.$emit('content-changed');
-      };
-    },
-    async onDismissSubmission(id) {
-      const piece = this.findDocById(this.items, id);
-      if (await this.dismissSubmission(piece)) {
-        apos.bus.$emit('content-changed');
-      };
-    },
-    async copy(id) {
-      apos.bus.$emit('admin-menu-click', {
-        itemName: `${this.moduleOptions.name}:editor`,
-        props: {
-          copyOf: this.findDocById(this.items, id)
-        }
-      });
-    },
-    // If pieceOrId is null, a new piece is created
-    async edit(pieceOrId) {
-      let piece;
-      if ((typeof pieceOrId) === 'object') {
-        piece = pieceOrId;
-      } else if (pieceOrId) {
-        piece = this.items.find(item => item._id === pieceOrId);
-      } else {
-        piece = null;
-      }
-      let moduleName;
-      // Don't assume the piece has the type of the module,
-      // this could be a virtual piece type such as "submitted-draft"
-      // that manages docs of many types
-      if (piece) {
-        if (piece.slug.startsWith('/')) {
-          moduleName = '@apostrophecms/page';
-        } else {
-          moduleName = piece.type;
-        }
-      } else {
-        moduleName = this.moduleName;
-      }
-      const doc = await apos.modal.execute(apos.modules[moduleName].components.editorModal, {
-        moduleName,
-        docId: piece && piece._id,
-        filterValues: this.filterValues
-      });
-      if (!doc) {
-        // Cancel clicked
-        return;
-      }
-      if (this.relationshipField) {
-        if (!this.checked.includes(doc._id)) {
-          doc._fields = doc._fields || {};
-          // Must push to checked docs or it will try to do it for us
-          // and not include _fields
-          this.checkedDocs.push(doc);
-          this.checked.push(doc._id);
-        }
-      }
-    },
     // Toolbar handlers
     archiveClick() {
```

Comment:
Is this obsolete?

Reply:
There was a partial story for this, the idea being it would be a batch operation with a more conspicuous button than others, but we can execute that later without a custom event just for this one percolating everywhere, so I mopped up.